### ISSUE TITLE
Find a place for "Your first pull request" tutorial

### ISSUE THREAD

**Issue by seanmadsen** [civicrm/civicrm-dev-docs#192]
TL;DR: Grab the content from [Your First Pull Request](https://github.com/civicrm/civicrm-buildkit/blob/d20229ac9f63aaf183687157a5e655c953b81f5e/doc/daily-coding.md#your-first-pull-request) and find a home for it in the Dev Guide.

I think it could potentially be incorporated into the [Contributing](https://docs.civicrm.org/dev/en/latest/core/contributing/) page by interspersing these commands into their relevant sections of that page.

Longer story: I'm migrating docs content out of the [buildkit repo](https://github.com/civicrm/civicrm-buildkit) and into the Dev Guide and as part of this process I'm deleting that [file](https://github.com/civicrm/civicrm-buildkit/blob/d20229ac9f63aaf183687157a5e655c953b81f5e/doc/daily-coding.md) which contains a mix of content that needs some work to be well-incorporated into the Dev Guide. I'm using issues to track the process of this migration, section-by-section, since I think the content will likely need to go to multiple places

**Comment by mattwire** [comment]
Add a bit on rebasing PRs so they only show relevant commits:
```git checkout branch_to_rebase
git fetch -f
git rebase -i origin/master  // Edit the commit description to remove commits you don't want
git push -f```

**Comment by KarinG** [comment]
@mattwire - thanks for rebasing this - looking at this now.

**Comment by KarinG** [comment]
Oops - wrong PR :-)
